src/include/sof/audio/component.h

```diff
@@ -233,12 +233,14 @@ enum {
 #endif /* #if defined(__ZEPHYR__) && defined(CONFIG_ZEPHYR_LOG) */
 
 #define comp_perf_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy peak plat %u cpu %u",		\
+	comp_info(comp_p, "perf_cycle comp_copy peak plat %u cpu %u",		\
 		  (uint32_t)((pcd)->plat_delta_peak),			\
 		  (uint32_t)((pcd)->cpu_delta_peak))
 
 #define comp_perf_avg_info(pcd, comp_p)					\
-	comp_info(comp_p, "perf comp_copy cpu avg %u (current peak %u)",\
+	comp_info(comp_p, "perf_cycle comp_copy samples %u period %u cpu avg %u  peak %u",\
```

Comment:
There's one extra blank in " peak". It's nicer for automatic parsing if there's no need to remove multiple successive blanks. It's good that you use only blanks as separators and removed the above () characters.